components/identities/actions/identityActions.tsx

```diff
@@ -172,6 +205,16 @@ const IdentityActions: FunctionComponent<IdentityActionsProps> = ({
                       icon="plus"
                       onClick={() => setIsSubdomainFormOpen(true)}
                     />
+                    {callDataEncodedDomain[0] === 1 ? (
+                      <ClickableAction
```

Comment:
<img width="399" alt="Capture d’écran 2023-05-28 à 19 00 20" src="https://github.com/starknet-id/app.starknet.id/assets/78437165/820df35e-795f-430c-be5e-ea5d7aa41fb5">

I want the enable auto-renewal to become the most important button (first and in green), the normal renewal comes in second and not in green.